.bazelrc

```diff
@@ -101,21 +101,20 @@ build:tsan-dev --test_env="TSAN_OPTIONS=report_atomic_races=0"
 build:release-common --define=no_debug_info=1
 
 # Compile releases optimizing for size (eg -Os, etc).
-build:release-common --config=sizeopt
+build:release-common -c opt --copt -Os
 
 # Set default symbols visibility to hidden to reduce .dynstr and the symbol table size
 build:release-common --copt=-fvisibility=hidden
 
 # Flags for release builds targeting iOS
 build:release-ios --config=ios
 build:release-ios --config=release-common
-build:release-ios --compilation_mode=opt
```

Comment:
This is now done with `-c opt` in `release-common`.

Reply:
mildly prefer reverting changes to this file as they're no longer necesary for bump but it's more of a style preference.